statuslistener/statuslistener.go

```diff
@@ -0,0 +1,153 @@
+package statuslistener
+
+import (
+	"sort"
+	"time"
+
+	c "github.com/RedHatInsights/sources-api-go/config"
+	"github.com/RedHatInsights/sources-api-go/dao"
+	"github.com/RedHatInsights/sources-api-go/internal/events"
+	"github.com/RedHatInsights/sources-api-go/kafka"
+	logging "github.com/RedHatInsights/sources-api-go/logger"
+	m "github.com/RedHatInsights/sources-api-go/model"
+	"github.com/RedHatInsights/sources-api-go/util"
+)
+
+const (
+	SourcesStatusTopic      = "platform.sources.status"
+	GroupID                 = "sources-api-status-worker"
+	EventAvailabilityStatus = "availability_status"
+)
```

Comment:
These can probably be un-exported, not sure why they'd need to be accessed from outside this package.